wiki/Skin/en.md

```diff
@@ -4,7 +4,29 @@ stub: true
 
 # Skin
 
-*See also: [Skinning](/wiki/Skinning)*\
-*For the dedicated sub-forum for skins, see [Skinning](https://osu.ppy.sh/community/forums/15)*
+A **skin** is a set of graphics (images) and custom sample sets (audio clips) that replace the default ones used by osu!. Many players regularly engage in the use of skins to allow for a more customised user experience, should they dislike the default skin. Almost all of what is visible or audible in osu! can be "skinned." 
 
-A **skin** is a set of graphics (images) and custom sample sets (audio clips) that replace the default ones used by osu!. Many players regularly engage in the use of skins to allow for a more customised user experience, should they dislike the default skin. Almost all of what is visible or audible in osu! can be "skinned."
+## Reasons to Use Skins
+
+- Skins can be used to further optimize a player's gameplay, since most skins do away with showing that a player has hit a note perfectly (to only show the minimum amount of information to a player in-game). 
```

Comment:
```suggestion
- Skins can be used to further optimize a player's gameplay, since many skins hide certain visual details or offer gameplay elements focused on a certain playstyle like specific mod combinations.
```